cypress/e2e/checkin/check-in-notes.cy.js

```diff
@@ -0,0 +1,123 @@
+      testData.defaultLocation = Location.getDefaultLocation(testData.servicePoint.id);
+      Locations.createViaApi(testData.defaultLocation).then((location) => {
+        InventoryInstances.createFolioInstancesViaApi({
+          folioInstances: testData.folioInstances,
+          location,
+        });
+      });
+      UserEdit.addServicePointViaApi(
+        testData.servicePoint.id,
+        userProperties.userId,
+        testData.servicePoint.id,
+      );
+      Checkout.checkoutItemViaApi({
+        id: uuid(),
+        itemBarcode: testData.folioInstances[0].barcodes[0],
+        loanDate: moment.utc().format(),
+        userBarcode: userData.barcode,
+        servicePointId: testData.servicePoint.id,
+      });
+      cy.getItems({
+        limit: 1,
+        expandAll: true,
+        query: `"barcode"=="${testData.folioInstances[0].barcodes[0]}"`,
+      }).then((res) => {
+        const itemData = res;
+        itemData.circulationNotes = [{ noteType: 'Check in', note: note1.title, staffOnly: true }];
+        cy.updateItemViaApi(itemData);
+      });
+      cy.login(userData.username, userData.password, {
+        path: TopMenu.checkInPath,
+        waiter: CheckInActions.waitLoading,
+      });
+      cy.getItems({
+        limit: 1,
+        expandAll: true,
+        query: `"barcode"=="${testData.folioInstances[0].barcodes[0]}"`,
+      }).then((res) => {
+        const itemData = res;
+        itemData.circulationNotes = [
+          ...itemData.circulationNotes,
+          { noteType: 'Check in', note: note2.title, staffOnly: true },
+        ];
+        cy.updateItemViaApi(itemData);
+      });
+    });
+  });
+
+  after('Deleting created entities', () => {
+    cy.getAdminToken();
+    CheckInActions.checkinItemViaApi({
+      itemBarcode,
+      servicePointId: testData.servicePoint.id,
+      checkInDate: new Date().toISOString(),
+    });
+    UserEdit.changeServicePointPreferenceViaApi(userData.userId, [testData.servicePoint.id]);
+    ServicePoints.deleteViaApi(testData.servicePoint.id);
+    InventoryInstances.deleteInstanceViaApi({
+      instance: testData.folioInstances[0],
+      servicePoint: testData.servicePoint,
+      shouldCheckIn: true,
+    });
+    Locations.deleteViaApi(testData.defaultLocation);
+    Users.deleteViaApi(userData.userId);
+  });
+
+  it(
+    'C776 Check in: check in notes (vega) (TaaS)',
+    { tags: [TestTypes.extendedPath, DevTeams.vega] },
+    () => {
+      // Scan item with at least two check in notes in Check In app
+      CheckInActions.checkInItemGui(itemBarcode);
+      // Check in note modal appears
+      CheckInModal.verifyModalTitle();
+      CheckInModal.checkNotes([note2, note1]);
+      // Click cancel on check in note modal
+      CheckInModal.closeModal();
+      // Scan item in Check In app
+      CheckInActions.checkInItemGui(itemBarcode);
+      // Check in note modal appears
+      CheckInModal.verifyModalTitle();
+      CheckInModal.checkNotes([note2, note1]);
```

Comment:
From test need to verify the date/time added, text of the note, and name of user who added the note (in format Lastname, Firstname) is displayed. User has option to cancel or confirm check in.